.github/workflows/test.yml

```diff
@@ -0,0 +1,62 @@
+name: Tests
+on: [pull_request]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+      - name: Install requirements
+        run: pip install flake8 pycodestyle
+      - name: Check syntax
+        run: |
+          flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+          flake8 . --count --max-line-length=127 --statistics
+
+
+  test:
+    needs: lint
+    name: CKAN 2.9-py2
+    runs-on: ubuntu-latest
+    container:
+      image: openknowledge/ckan-dev:2.9-py2
+    services:
+      solr:
+        image: ckan/ckan-solr-dev:2.9-py2
+      postgres:
+        image: ckan/ckan-postgres-dev:2.9-py2
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+      redis:
+          image: redis:3
+    env:
+      CKAN_SQLALCHEMY_URL: postgresql://ckan_default:pass@postgres/ckan_test
+      CKAN_DATASTORE_WRITE_URL: postgresql://datastore_write:pass@postgres/datastore_test
+      CKAN_DATASTORE_READ_URL: postgresql://datastore_read:pass@postgres/datastore_test
+      CKAN_SOLR_URL: http://solr:8983/solr/ckan
+      CKAN_REDIS_URL: redis://redis:6379/1
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install requirements
+      run: |
+        pip install -r requirements.txt
+        pip install -r dev-requirements.txt
+        pip install -e "git+https://github.com/fjelltopp/ckanext-scheming@development#egg=ckanext-scheming"
+        pip install -r https://raw.githubusercontent.com/fjelltopp/ckanext-scheming/development/requirements.txt
+
+        pip install ckantoolkit
```

Comment:
This should already be installed in validation requirements.txt
```
ckantoolkit>=0.0.3
```

Reply:
right, didn't notice that :)